code/__DEFINES/machines.dm

```diff
@@ -29,6 +29,7 @@
 #define BIOGENERATOR 32 //Uses biomass
 #define LIMBGROWER 64 //Uses synthetic flesh
 #define SMELTER 128 //uses various minerals
+#define	AUTOYLATHE	256	//Uses glass/metal/Plastic

```

Comment:
keep indents/spacing constant. look at the one above.

Reply:
Shit. Fixing that.